homework-tips.md

```diff
@@ -0,0 +1,83 @@
+# Tips on Submitting Your Homework
```

Comment:
Having some screenshots in this section might be helpful to folks unfamiliar with the Github UI.

Reply:
I created one.